src/sharedInterfaces/schemaCompare.ts

```diff
@@ -33,11 +34,15 @@ export const enum SchemaCompareEndpointType {
     // located at \src\Microsoft.SqlTools.ServiceLayer\SchemaCompare\Contracts\SchemaCompareRequest.cs
 }
 
+export interface ISchemaCompareConnectionProfile extends IConnectionInfo {
```

Comment:
Can you reuse `IConnectionDialogProfile` instead since that's already sorta playing this role?